config/helm/crd_core_cainjection.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Patch for setting our cert-manager.io/inject-ca-from annotation (overriding namespace part) in all core CRDs (label selector is used in kustomization.yaml to apply this to all core CRDs).